Makefile

```diff
@@ -149,7 +163,11 @@ update-deps-overrides:
 	cp "$(tmp_dir)"/{updated,current}/deps.diff
 .PHONY: update-deps-overrides
 
-verify: verify-govet verify-gofmt
+verify-helm:
+	@$(foreach chart,$(HELM_CHARTS),helm lint helm/$(chart); )
```

Comment:
this will ignore errors
```
helm lint helm/chart_0; helm lint helm/chart_1; helm lint helm/chart_2; 
```

Reply:
if you are suppressing the execution output, might be nice to at least have an echo saying you are linting

Reply:
Helm itself is verbose about it